src/js/helpers/Util.js

```diff
@@ -0,0 +1,15 @@
+/*eslint-disable no-alert */
```

Comment:
@rcorral this should be in the direction you suggested. Of course it's supposed to be transitional - the goal is to get rid of alerts and company altogether.